### PR TITLE
SSD python example

### PR DESCRIPTION
**Pull request by wzhongyuan**
Add ssd python example

### CODE REVIEW

File: models/src/main/python/bigdl/zoo/example/objectDetection/Predict.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+JavaCreator.set_creator_class("com.intel.analytics.zoo.models.pythonapi.PythonModels")
+sc = get_spark_context(conf=create_spark_conf())
+init_engine()
+
+parser = argparse.ArgumentParser()
+parser.add_argument('modelPath', help="Path where the model is stored")
+parser.add_argument('imgPath', help="Path where the images are stored")
+parser.add_argument('outputPath',  help="Path to store the detection results")
+
+def predict(modelPath, imgPath, outputPath):
```
Comment (SeaOfOcean) [inline models/src/main/python/bigdl/zoo/example/objectDetection/Predict.py]: model_path, img_path, output_path, etc, follow python coding style

File: models/src/main/python/bigdl/zoo/example/objectDetection/Predict.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    model = Model.loadModel(modelPath)
+    imageFrame = ImageFrame.read(imgPath, sc)
```
Comment (SeaOfOcean) [inline models/src/main/python/bigdl/zoo/example/objectDetection/Predict.py]: image_frame

File: models/src/main/python/bigdl/zoo/example/objectDetection/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+2. Setup environment
+
+Add model jar to `BIGDL_CLASSPATH` and python zip file to `PYTHONPATH`
+
+export DL_PYTHON_HOME=${path_to_analytics-zoo}/models/target/bigdl-models-0.1-SNAPSHOT-python-api.zip
+
+export PYTHONPATH=$PYTHONPATH:${path_to_bigdl}/dist/conf/spark-bigdl.conf:$DL_PYTHON_HOME
+
+export BIGDL_CLASSPATH=$BIGDL_CLASSPATH:${path_to_analytics-zoo}/models/target/models-0.1-SNAPSHOT-jar-with-dependencies.jar
```
Comment (SeaOfOcean) [inline models/src/main/python/bigdl/zoo/example/objectDetection/README.md]: what's the usage of BIGDL_CLASSPATH here?

File: models/src/main/python/bigdl/zoo/example/objectDetection/README.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
+## Object Detection Python example
+This example illustrates how to detect objects given image and model
+### Run steps
+1. Build 
+
+./build.sh
```
Comment (SeaOfOcean) [inline models/src/main/python/bigdl/zoo/example/objectDetection/README.md]: surround with code block

```
./build.sh
```

File: models/src/main/python/bigdl/zoo/example/objectDetection/README.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+### Run steps
+1. Build 
+
+./build.sh 
+
+2. Setup environment
+
+Add model jar to `BIGDL_CLASSPATH` and python zip file to `PYTHONPATH`
+
+export DL_PYTHON_HOME=${path_to_analytics-zoo}/models/target/bigdl-models-0.1-SNAPSHOT-python-api.zip
```
Comment (SeaOfOcean) [inline models/src/main/python/bigdl/zoo/example/objectDetection/README.md]: surround with code block

File: models/src/main/python/bigdl/zoo/example/objectDetection/README.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+export PYTHONPATH=$PYTHONPATH:${path_to_bigdl}/dist/conf/spark-bigdl.conf:$DL_PYTHON_HOME
+
+export BIGDL_CLASSPATH=$BIGDL_CLASSPATH:${path_to_analytics-zoo}/models/target/models-0.1-SNAPSHOT-jar-with-dependencies.jar
+
+3. Run the example
+
+master=... // spark master
```
Comment (SeaOfOcean) [inline models/src/main/python/bigdl/zoo/example/objectDetection/README.md]: surround with code block

File: models/src/main/python/bigdl/zoo/example/objectDetection/README.md (hunk 3 above)
Comment (wzhongyuan) [reply models/src/main/python/bigdl/zoo/example/objectDetection/README.md]: could be removed

**Comment by wzhongyuan** [comment]
@SeaOfOcean  refined per review comments

**Comment by wzhongyuan** [comment]
Closing this as already get reviewed

**Pull request merged**